common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -735,9 +739,9 @@ def ignore_conditions_table
         # Return an empty string if no valid ignore conditions after filtering
         return "" if valid_ignore_conditions.empty?
 
-        # Sort them by updated_at (or created_at if updated_at is nil), taking the latest 20
+        # Sort them by updated_at, taking the latest 20
         sorted_ignore_conditions = valid_ignore_conditions.sort_by do |ic|
-          ic["updated_at"].nil? ? T.must(ic["created_at"]) : T.must(ic["updated_at"])
+          ic["updated-at"].nil? ? Time.at(0).iso8601 : T.must(ic["updated-at"])
```

Comment:
I mentioned in the PR body, only `updated-at` is sent in the job payload, not `created-at` nor `created_at`.

Reply:
Sorry I missed that; please do you know why that is the case? Why does the job not send that info even if the code consumes it

Reply:
>Why does the job not send that info even if the code consumes it.

In Rails when a record is created, it sets both columns `updated_at` and `created_at` to `Time.now` automagically for you, so it's redundant to send both. See the discussion on PR that added updated_at: https://github.com/github/dependabot-api/pull/4319#pullrequestreview-1567173558

I'm unsure why `created_at` was added in #9130, @JamieMagee was there something that made you think this data was being sent in the job payload?

> What would it take to explore your suggestion?

I filed a tech debt issue for this: https://github.com/dependabot/dependabot-core/issues/9561

It might take a few hours to sort out all the places where a hash is used and turn it into a class instance, but I think it would be worth the effort.

Reply:
Thank you so much!

Reply:
I assume @JamieMagee put it in cos of this comment:

>         # Sort them by updated_at (or created_at if updated_at is nil), taking the latest 20